src/test/java/com/ibm/watson/developer_cloud/service/GenericServiceTest.java

```diff
@@ -238,5 +302,57 @@ public void testCustomUserAgent() throws InterruptedException {
     assertTrue(request.getHeader(HttpHeaders.USER_AGENT).endsWith("foo-bar"));
     service.setDefaultHeaders(null);
   }
-  
+
+  /**
+   * Tests request without authentication.
+   */
+  @Test
+  public void testNoAuthentication() {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://www.codacy.com/app/watson-developer-cloud/java-sdk/file/2722996208/issues/source?bid=3303699&fileBranchId=3347904#l310)